controllers/provider-openstack/pkg/webhook/controlplanebackup/ensurer.go

```diff
@@ -110,57 +110,21 @@ func (e *ensurer) getBackupRestoreContainer(name string, cluster *extensionscont
 			provider = openstack.StorageProviderName
 			env = []corev1.EnvVar{
 				{
-					Name: "STORAGE_CONTAINER",
+					Name: "OS_CLIENT_CONFIG_FILE",
```

Comment:
You will still have to pass bucketName via STORAGE_CONTAINER env.

Reply:
Good point.